<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>601</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>RPG_Peptide-Selection</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
border: 2px solid rgb(120, 120, 120);
}

*{
background-color: rgb(40,40, 32);
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <widget class="QWidget" name="tab" native="true">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-bottom: 2px solid rgb(120, 120, 120);</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item alignment="Qt::AlignRight|Qt::AlignTop">
       <widget class="QFrame" name="tab_bar_widget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>12000</width>
          <height>30</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-bottom: none;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="minimizeButton">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="toolTip">
            <string>Minimize Window</string>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton:hover{
	
	
	icon: url(:/icons-hover/icons-hover/minus.svg);
	
	
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../MainWin/resources.qrc">
             <normaloff>:/icons/icons/minus.svg</normaloff>:/icons/icons/minus.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>18</width>
             <height>18</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="xButton">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="toolTip">
            <string>Close Window</string>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton:hover{
	
	
	icon: url(:/icons-hover/icons-hover/x.svg);
	
	
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../MainWin/resources.qrc">
             <normaloff>:/icons/icons/x.svg</normaloff>:/icons/icons/x.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>7</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Eight Bit Dragon</family>
           <pointsize>15</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 15pt &quot;Eight Bit Dragon&quot;;
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Insert  Length of your Peptide</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>7</number>
         </property>
         <property name="bottomMargin">
          <number>10</number>
         </property>
         <item>
          <widget class="QLineEdit" name="lineEdit">
           <property name="minimumSize">
            <size>
             <width>320</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>320</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::StrongFocus</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 8pt &quot;Eight Bit Dragon&quot;;
color: rgb(255, 255, 255);
border: 1px solid white;
border-radius: 10px;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>Insert any number between 2-12</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="generateButton">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
font: 10pt &quot;Eight Bit Dragon&quot;;
color: rgb(255, 255, 255);
border: 2px solid black;
background-color: rgb(40,40, 32);
border-color: rgb(100, 100, 100);
border-radius: 12px;
}
QPushButton:hover{
color: rgb(255, 255, 255);
border: 1px solid;
background-color: rgb(80, 80, 80);
border-color: rgb(64, 64, 64);
border-radius: 12px;
}</string>
           </property>
           <property name="text">
            <string>Generate</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="Button">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>185</green>
             <blue>139</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>185</green>
             <blue>139</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Window">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>185</green>
             <blue>139</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="Button">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>185</green>
             <blue>139</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>185</green>
             <blue>139</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Window">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>185</green>
             <blue>139</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="Button">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>185</green>
             <blue>139</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>185</green>
             <blue>139</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Window">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>185</green>
             <blue>139</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="tabletTracking">
        <bool>false</bool>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::DefaultContextMenu</enum>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="toolTipDuration">
        <number>-2</number>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QTabBar::tab {
	background-color: rgb(40,40, 32);
	color: white;
	font: 8pt &quot;Eight Bit Dragon&quot;;
	border: 2px solid black;
	border-color: rgb(64, 64, 64);
	border-top-left-radius: 8px;
	border-top-right-radius: 8px;
	width: 30px;
	height: 30px;
	

	
}
QTabBar::tab::hover {
	background-color: rgb(80, 80, 80);
	color: white;
	font: 8pt &quot;Eight Bit Dragon&quot;;
	border: 2px solid black;
	
	
	
}

QTabBar::tab::selected {

	background-color: rgb(255, 129, 60);
	color: white;
	font: 8pt &quot;Eight Bit Dragon&quot;;
	border: 2px solid black;
	
	
	
}

*{
border: none;
font: 10pt &quot;Eight Bit Dragon&quot;;
background-color: rgb(255, 185, 139);
border-top-left: 1px solid white;
}

</string>
       </property>
       <property name="tabPosition">
        <enum>QTabWidget::North</enum>
       </property>
       <property name="tabShape">
        <enum>QTabWidget::Rounded</enum>
       </property>
       <property name="currentIndex">
        <number>-1</number>
       </property>
       <property name="usesScrollButtons">
        <bool>false</bool>
       </property>
       <property name="documentMode">
        <bool>true</bool>
       </property>
       <property name="tabsClosable">
        <bool>false</bool>
       </property>
       <property name="tabBarAutoHide">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="selectallLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="widget_2" native="true">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">
background-color: rgb(255, 185, 139);</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
         <widget class="QPushButton" name="selectButton">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Eight Bit Dragon</family>
            <pointsize>8</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
font: 8pt &quot;Eight Bit Dragon&quot;;
color: rgb(255, 255, 255);
border: 2px solid black;
background-color: rgb(40,40, 32);
border-color: rgb(100, 100, 100);
border-radius: 12px;
}
QPushButton:hover{
color: rgb(255, 255, 255);
border: 1px solid;
background-color: rgb(80, 80, 80);
border-color: rgb(64, 64, 64);
border-radius: 12px;
}</string>
          </property>
          <property name="text">
           <string>Select All</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QPushButton" name="listoButton">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
font: 10pt &quot;Eight Bit Dragon&quot;;
color: rgb(255, 255, 255);
border: 2px solid black;
background-color: rgb(40,40, 32);
border-color: rgb(100, 100, 100);
border-radius: 12px;
}
QPushButton:hover{
color: rgb(255, 255, 255);
border: 1px solid;
background-color: rgb(80, 80, 80);
border-color: rgb(64, 64, 64);
border-radius: 12px;
}</string>
       </property>
       <property name="text">
        <string>Ready!</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QPushButton" name="resetButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>40</height>
           </size>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
font: 10pt &quot;Eight Bit Dragon&quot;;
color: rgb(255, 255, 255);
border: 2px solid black;
background-color: rgb(40,40, 32);
border-color: rgb(100, 100, 100);
border-radius: 12px;
}
QPushButton:hover{
color: rgb(255, 255, 255);
border: 1px solid;
background-color: rgb(80, 80, 80);
border-color: rgb(64, 64, 64);
border-radius: 12px;
}</string>
          </property>
          <property name="text">
           <string>Reset</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="resetButton_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>40</height>
           </size>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
font: 10pt &quot;Eight Bit Dragon&quot;;
color: rgb(255, 255, 255);
border: 2px solid black;
background-color: rgb(40,40, 32);
border-color: rgb(100, 100, 100);
border-radius: 12px;
}
QPushButton:hover{
color: rgb(255, 255, 255);
border: 1px solid;
background-color: rgb(80, 80, 80);
border-color: rgb(64, 64, 64);
border-radius: 12px;
}</string>
          </property>
          <property name="text">
           <string>Check</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../MainWin/resources.qrc"/>
 </resources>
 <connections/>
</ui>
